.gitignore

```diff
@@ -9,3 +9,4 @@
 
 # rspec failure tracking
 .rspec_status
+Gemfile.lock
```

Comment:
Why are we ignoring this?

Reply:
because it's a gem

Reply:
https://github.com/rootstrap/tech-guides/blob/master/open-source/developing_gems.md
https://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/

Reply:
in other words, a gem should work with several versions of gems and ruby, a Gemfile.lock would mean the gem works only with that specific versions